src/main/java/ai/elimu/rest/v2/crowdsource/WordContributionRestController.java

```diff
@@ -156,7 +156,7 @@ public String postWordContribution(
             wordContributionEvent.setTime(wordContributionEventGson.getTime());
             wordContributionEvent.setWord(word);
             wordContributionEvent.setRevisionNumber(word.getRevisionNumber());
-            wordContributionEvent.setComment(wordContributionEventGson.getComment());
+            wordContributionEvent.setComment(StringUtils.abbreviate(wordContributionEventGson.getComment(), 1000));
```

Comment:
@arvind-1 Thanks for remembering to include the change for the REST controller, I forgot about that. 🙂

Can you also implement the same change in the `WordCreateController` and `WordEditController`?